File: ex07.py
def string_test(s):
    d={"upper":0, "lower":0}
    for c in s:
        if c.isupper():
           d["upper"]+=1
        elif c.islower():
           d["lower"]+=1
        else:
           pass
    print ("Original String: ", s)
    print ("number of upper case letters: ", d["upper"])
    print ("numbers of lower case letters: ", d["lower"])

string_test('My name is Xochitl Sosa')

File: ex07.cpp
#include<iostream> 
using namespace std; 
  
void Count(string str) 
{ 
    int upper = 0, lower = 0; 
    for (int i = 0; i < str.length(); i++) 
    { 
        if (str[i] >= 'A' && str[i] <= 'Z') 
            upper++; 
        else if (str[i] >= 'a' && str[i] <= 'z') 
            lower++; 
        else if (str[i]>= '0' && str[i]<= '9')
        ; 
           
    } 
    cout << "upper case letters: " << upper << endl; 
    cout << "lower case letters : " << lower << endl; 
   
} 
  
// Driver function 
int main() 
{ 
    string str = "Mi nombre es Xochitl Sosa"; 
    Count(str); 
    return 0; 
} 